habitat/tasks/utils.py

```diff
@@ -5,7 +5,6 @@
 # LICENSE file in the root directory of this source tree.
 
 import numpy as np
-import quaternion
```

Comment:
This import should stay as you can't be sure that numpy will be patched and `np.quaternion` will exist otherwise.

Reply:
Nice catch!.